restore/action.yml

```diff
@@ -11,6 +11,10 @@ inputs:
   restore-keys:
     description: 'An ordered list of keys to use for restoring stale cache if no cache hit occurred for key. Note `cache-hit` returns false in this case.'
     required: false
+  enableCrossOsArchive:
+    description: 'An optional boolean enabled to restore cache on windows which could be saved on any platform'
```

Comment:
```suggestion
    description: 'An optional boolean to enable restoring a cache on windows which was created on another platform'
```

Reply:
Would you take a look at above comment by Lovepreet as well?

Reply:
replied.